extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -220,15 +220,27 @@ const OHIFCornerstoneViewport = React.memo(props => {
     const currentPresentation = cornerstoneViewportService.getPresentation(
       viewportIndex
     );
-    const { presentationSync } = stateSyncService.getState();
-    if (currentPresentation) {
-      stateSyncService.store({
-        presentationSync: {
-          ...presentationSync,
-          [currentPresentation.id]: currentPresentation,
-        },
-      });
+    if (!currentPresentation || !currentPresentation.presentationIds) return;
+    const {
+      lutPresentationSync,
+      positionPresentationSync,
```

Comment:
the names here are werid to me, since we have `lutPresentationId` and `positionPresentationId` below as well. and there are `lutPresentationSync` and `positioinPresentationSync` too .....

Reply:
`lutPresentationStore` and `positionPresentationStore`?